maquetado/scss/login.scss

```diff
@@ -0,0 +1,56 @@
+.login {
+  width: 300px;
+  height: 400px;
```

Comment:
You don't need to set a height here. Your content will define the component's height.